google/resource_container_cluster_test.go

```diff
@@ -1340,3 +1395,49 @@ resource "google_container_cluster" "with_node_pool_node_config" {
 }
 `, testId, testId)
 }
+
+func testAccContainerCluster_withIPAllocationPolicy(ranges, policy map[string]string) string {
+
+	var secondaryRanges bytes.Buffer
+	for rangeName, cidr := range ranges {
+		secondaryRanges.WriteString(fmt.Sprintf(`
+	secondary_ip_range {
+	    range_name    = "%s"
+	    ip_cidr_range = "%s"
+	}`, rangeName, cidr))
+	}
+
+	var ipAllocationPolicy bytes.Buffer
```

Comment:
I'm fine leaving this as is, but a potential alternative would be just passing in the configuration block as a string that looked like:
```
  ip_allocation_policy {
    use_ip_aliases                = "true"
    cluster_secondary_range_name  = "pods"
    services_secondary_range_name = "services"
  }
```